src/Components/RevenueBonusArea.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import ToggleButton from './ToggleButton';
+import { ColorsToggleButton } from './ToggleButton';
+import { HouseDemandContext } from './HouseDemandContext';
+import Title from './Title';
+
+type RevenueBonusProps = {};
+
+const RevenueBonusArea: React.FC<RevenueBonusProps> = () => {
+  const { hasGarden, setHasGarden, hasCFO, setHasCFO, } = useContext(HouseDemandContext);
+
+  const handleHasGarden = () => {
+    setHasGarden(!hasGarden);
+  };
+
+  const handleHasCFO = () => {
+    setHasCFO(!hasCFO);
+  };
+
+  return (
+    <section>
+      <Title>Revenue Bonuses</Title>
+      <div>
+        <ToggleButton
+          label="Garden"
+          active={hasGarden}
+          onClick={handleHasGarden}
+        />
+        <ToggleButton
+          label="CFO"
+          active={hasCFO}
+          onClick={handleHasCFO}
+          activeColor={ColorsToggleButton.purple}
+        />
```

Comment:
Based on the screenshot we potentially need a minimum width or playing around with `display: grid` or something. min-width is definitely a lot easier here. Just looks a little weird being so thin.

Reply:
As above, will style with waitresses added since that's going to then look a bit different and grid might make more sense.